pkg/storage/cacher_test.go

```diff
@@ -236,7 +237,7 @@ func TestWatch(t *testing.T) {
 	}
 	defer initialWatcher.Stop()
 
-	verifyWatchEvent(t, initialWatcher, watch.Added, podFoo)
+	t.Log("verifying")
```

Comment:
I would prefer leaving the test exactly what it was testing before your change.
So what we would like to do is to pass "fooCreated.ResourceVersion - 1" instead in line 234?